src/status_im/contexts/wallet/add_account/create_account/import_private_key/view.cljs

```diff
@@ -108,26 +116,27 @@
                                 (i18n/label :t/import-private-key-info)])
                              [quo/button
                               {:customization-color customization-color
-                               :disabled?           (or (empty? input-value) error?)
-                               :on-press            #()}
+                               :disabled?           (or (string/blank? private-key) error?)
+                               :on-press            #(rf/dispatch [:navigate-to
+                                                                   :screen/wallet.keypair-name
+                                                                   {:workflow :import-private-key}])}
```

Comment:
Tiny comment 🌱

Because the `on-press` callback does not need `use-callback`, perhaps we can extract this anonymous function into a static function outside of the component.

For example, something like this:
```clojure
(defn navigate-to-keypair-import
  []
  (rf/dispatch [:navigate-to
                :screen/wallet.keypair-name
                {:workflow :import-private-key}]))
```

Reply:
Fixed, with couple other callbacks =)